packages/sources/bitgo/src/transport/wallet.ts

```diff
@@ -0,0 +1,230 @@
+import { TransportDependencies } from '@chainlink/external-adapter-framework/transports'
+import { ResponseCache } from '@chainlink/external-adapter-framework/cache/response'
+import { Requester } from '@chainlink/external-adapter-framework/util/requester'
+import { AdapterResponse, makeLogger, sleep } from '@chainlink/external-adapter-framework/util'
+import { SubscriptionTransport } from '@chainlink/external-adapter-framework/transports/abstract/subscription'
+import { EndpointContext } from '@chainlink/external-adapter-framework/adapter'
+import { BaseEndpointTypes, inputParameters } from '../endpoint/wallet'
+import { AdapterError } from '@chainlink/external-adapter-framework/validation/error'
+import { getApiInfo } from './utils'
+
+const logger = makeLogger('BitgoTransport')
+
+type RequestParams = typeof inputParameters.validated
+
+export type WalletTransportTypes = BaseEndpointTypes
+
+interface WalletResponse {
+  wallets: {
```

Comment:
are we guaranteed to have wallets in this array or could it be empty?

Reply:
I think being empty is a valid case

Reply:
not convinced this would be an issue, so let's just note and fix in a follow up if we notice any weirdness